.github/workflows/publish.yml

```diff
@@ -43,7 +43,8 @@ jobs:
       - name: Set target repo checkout branch
         # Note: Branches registered here MUST BE protected in the target repo!
         if: |
-          fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target'
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target' ||
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && (fromJSON(steps.inputs.outputs.result).merge_target == 'v7' || fromJSON(steps.inputs.outputs.result).merge_target == 'master')
```

Comment:
master should not be needed -- that's the default branch

Reply:
`master` is not the default branch in the JS repo, it's `develop` (outlined my reasoning in the PR description).

Reply:
I'd rather leave `master` out for now

Reply:
Do you have a specific concern with opting in `master`? I just checked and branch protection rules are set for both `master` and `develop` if that's the concern. Additionally to the reason in the PR description, I'd like to avoid a situation in which we use two different strategies for the config within one repo.

Reply:
usually I like to keep changes entirely separated -- adding v7 and adding master are two separate concerns.

it helps when needing to revert as well and it keeps things focused

when we sat down and designed this we all agreed on `v7` as something to add here, but `master` is scope creep that I haven't had time to consider all of the same cases for.  I'd like to not block your moves for `v7` but also make sure I've done the due dilligence for `master` instead of it being slipped in right at the end.

Reply:
ok fair enough, I'll split this PR and open a separate one for `master`. Thanks!